late/scripts/chroot.sh

```diff
@@ -88,12 +88,22 @@ if ! mount | grep "$TARGET/sys"; then
     mount -t sysfs targetsys $TARGET/sys
     MOUNT_CLEANUP="$TARGET/sys $MOUNT_CLEANUP"
 fi
+if ! mount | grep "$TARGET/sys/firmware/efi/efivars"; then
+    mount -t efivarfs efivarfs $TARGET/sys/firmware/efi/efivars
+    MOUNT_CLEANUP="$TARGET/sys/firmware/efi/efivars $MOUNT_CLEANUP"
+fi
```

Comment:
I don't see any use of `mokutil --test-key`.  Is that coming in a later PR? Or if it's dropped this section should be dropped too.

Reply:
mokutil --test-key will be put another fish script which will be done in stage2 in chroot environment. So, this PR is just to provide an environment that fish script can do more operation in chroot environment.

Reply:
the idea is that, in the end of stage 2, fish script still need to check if the mok key have been enrolled, and enroll it if --test-key return false.

Reply:
OK.